testing-modules/junit-5/src/test/java/com/baeldung/execution/time/SampleExecutionTimeUnitTest.java

```diff
@@ -0,0 +1,46 @@
+package com.baeldung.execution.time;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class SampleExecutionTimeUnitTest {
```

Comment:
If this is a JUnit 5 test, then the class and the methods should not be marked `public`